htdocs/kernel/menusitems.php

```
@@ -0,0 +1,263 @@
+    }
+
+    /**
+     * @return mixed
+     */
+    public function get_new_enreg()
+    {
+        global $xoopsDB;
+        $new_enreg = $xoopsDB->getInsertId();
+
+        return $new_enreg;
+    }
```

PHPMD (2.15.0)

[error] The method get_new_enreg is not named in camelCase. (undefined)

(CamelCaseMethodName)